app/models/worklog_report.rb

```diff
@@ -327,7 +327,7 @@ def key_from_worklog(w, r)
       "6_approved"
     elsif (property = Property.find_by_filter_name(current_user.company, r))
       w.task.property_value(property)
-    elsif r and (match = r.match(/ca_(\d+)/))
+    elsif r and (r.match(/ca_(\d+)/))
```

Comment:
Check for custom attributes id and return string with ca_id (custom attribute id)